src/promptflow-devkit/promptflow/_sdk/_orchestrator/run_submitter.py

```diff
@@ -96,13 +95,13 @@ def _run_bulk(self, run: Run, stream=False, **kwargs):
             # pass with internal parameter `_collection`
             start_trace(
                 attributes=attributes,
-                run=run.name,
+                run=run,
```

Comment:
You changed the parameter type from `str` to `Run` object, have you double checked all inside usage are compatible?

Reply:
i understand `start_trace` cannot well handle that `run` is a `Run` object, please verify this change

Reply:
I made some type judgments where I used it:
![image](https://github.com/microsoft/promptflow/assets/23182548/551d7aa8-3bc8-4a12-a9a5-0e030872887c)